docs/scripting-ref.rst

```diff
@@ -331,6 +331,32 @@ Compare:
 See also: :ref:`splash-runjs`, :ref:`splash-jsfunc`,
 :ref:`splash-wait-for-resume`, :ref:`splash-autoload`.
 
+.. _splash-disablejs:
+
+splash:disablejs
```

Comment:
That's a good question. We have 3 options:

1. splash:enablejs() / splash:disablejs()
2. splash:set_js_enabled(...)
3. splash.js_enabled = ...

I proposed (1) because it looks nicer in user scripts than (2) - splash:set_js_enabled(false) is longer and less readable.

There is indeed splash:set_images_enabled already, and it is better to be consistent though.

(3) also provides a nice API (even nicer). It is shorter and allows to read the value as well as write to it. A problem with (3) is that it is a bit harder to implement (sandbox code should be changed), and that it is harder to indicate errors or make action async this way. There shouldn't be any errors, and the operation is not async, so (3) looks good to me.